user/models.py

```diff
@@ -27,6 +36,30 @@ def is_editor(self):
     def is_editor(self, value):
         pass
 
+    @property
+    def role(self):
+        """Returns the highest role that a user has."""
+        if self.is_staff:
+            return "Staff"
+        elif self.groups.filter(name="Editor").exists():
+            return "Editor"
+        elif self.groups.filter(name="Copyeditor").exists():
+            return "Copyeditor"
+        else:
+            return "Contributor"
+
+    @role.setter
+    def role(self, value):
```

Comment:
From what I can tell, using this setter won't actually persist anything.

Reply:
Behaviour of role has been changed.